daylight_saving.md

```diff
@@ -0,0 +1,59 @@
+NTFS Timezones and Daylight saving
+=================
+
+## Handling time-zones
+
+The `network_timezone` field in file networks.txt sets the time zone for all `stop_times` of `trips` of the `network`. 
+
+If a feed covers multiple time zones, a specific time zone information can be specified on a `stop_point` outside the `network_timezone` in the `stop_timezone` field. Set this optional field in file `stops.txt` (for stops with location_type = 0). If not set, the timezone of the network is used. Ensure that trip travel times remain accurate and consistent by providing departure and arrival times based on the `network_timezone` for stops that are out of that timezone.
+
+Navitia presents time for departure and arrival times in the local time zone of each stop. \
+Warnings: 
+- Time in `stop_times.txt` should always increase along a trip. 
+- NTFS uses the local timezone at noon of the travel day of the trip. This is the implementation of the constraint specified on the (GTFS Time data format)[http://gtfs.org/schedule/reference/#field-types]:
+> Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. \
+Example: 14:30:00 for 2:30PM or 25:35:00 for 1:35AM on the next day.
+
+
+## Daylight Saving Time
+Some trips may operate during the switch from or to Daylight Saving time and then continue to operate as normal. The NTFS format uses the same way of dealing with daylight saving than [the GTFS does](https://support.google.com/transitpartners/answer/7074707).
+It is strongly recommended to explicitly define trips operating across time changes (not using multiple day calendar).
```

Comment:
I'm trying to say that dealing with time changes can be done, but with trips associated with only one active day in the associated calendar. Do you have a better way of explaining  this?

Reply:
I think I understand now (I was not when I did my previous comment). I'll try some ideas below, but indeed, not an easy recommendation to convey.

> It is recommended to define a trip operating across a time change, on a single service day (see `calendar.txt`).

> It is recommended to avoid services on multiple days (see `calendar.txt`) for trips that cross time changes.

Note that I removed the word `strongly` because it's already a recommendation, `strongly` only make it subjective instead of objective.

Also note that I added a mention of `calendar.txt` because this sentence either contains calendar day, or service day, but both can be understood as jargon... [or not](https://www.wikiwand.com/en/Calendar_day). Mentioning `calendar.txt` (which itself has a `service_id`) makes it obvious we're using the jargon here, not just any ~calendar day~ day of the week.

Reply:
Thanks for the proposal. I've chosen the 2nd one. Should be good now.